tests/functional/afr/test_replica_eager_lock.py

```diff
@@ -0,0 +1,184 @@
+#  Copyright (C) 2017-2021 Red Hat, Inc. <http://www.redhat.com>
+#
```

Comment:
I think my previous comment wasn't clear, @srivickynesh you just need `2021`.

Reply:
done